Framework/PythonInterface/test/python/plugins/algorithms/WorkflowAlgorithms/IndirectReplaceFitResultTest.py

```diff
@@ -25,23 +27,64 @@ def execute_algorithm(input_workspace, single_fit_workspace, output_name):
                              OutputWorkspace=output_name)
 
 
+def get_sequential_input_string(workspace_name, number_of_histograms):
+    input_string = ''
+    for i in range(number_of_histograms):
+        input_string += '{0},i{1};'.format(workspace_name, i)
+    return input_string
+
+
+def current_os_has_gslv2():
+    """ Check whether the current OS is using GSLv2 """
+    return platform.linux_distribution()[0].lower() == 'ubuntu' or platform.mac_ver()[0] != ''
```

Comment:
I feel this is a check other techniques must require in the code at one point or another. It's okay here for this PR, but it may be worth checking if this can be moved to a general mantid method to clean up the codebase slightly
@martyngigg

Reply:
@TTitcombe You are correct. There is a maintenance task to add a `buildconfig.py` module to `mantid` so that it can be queried for things like this rather than relying on checking the OS etc.